stories/example-screen-staff-list.stories.js

```diff
@@ -27,150 +17,1023 @@ export default {
+                    Home
+                  </span>
+                </a>
+              </li>
+            </ul>
+            <div class="ps-4 pe-4 pt-6 pb-6 my-2">
+              <div class="dropdown">
+                <div class="d-flex align-items-center flex-wrap btn-search btn btn-action btn-action-icon-only btn-action-header w-100" type="button" id="dropdownMenuButton2" data-bs-toggle="dropdown" aria-expanded="false" style="background-color: #ECECEC;">
+                  <svg class="icon-xl-font-size mx-2">
+                    <use xlink:href="${iconSprite}#tabler-search" />
+                    <span class="visually-hidden">Open Search Dialog</span>
+                  </svg>
+                </div>
+              </div>
+            </div>
+            <div class="d-flex flex-column justify-content-start ps-4 pe-4 pt-6 pb-6">
+              <button type="button" class="btn btn-action btn-action-icon-only btn-action-header rounded-circle">
+                <svg class="icon-lg-font-size mt-n1">
+                  <use xlink:href="${iconSprite}#tabler-adjustments-horizontal" />
+                  <span class="visually-hidden">Thumbs Down</span>
+                </svg>
+              </button>
+              <button class="avatar avatar-md avatar-image rounded-circle mt-2" type="button">
+                <img role="button" class="" src="https://www.edge.org/sites/default/files/styles/member-photo/public/member-pictures/picture-510-1416068804.jpg?itok=cnAQyDr5" alt="...">
+              </button>
+            </div>
+          </nav>
         </div>
-      </nav>
+      </div>
     </header>
-  `;
-};
+  </div>
+`
+
+const tabsDoubleRowHTML = `
+  <ul class="nav nav-tabs bg-white rounded-top border border-light ps-7 pe-7">
+    <li class="nav-item">
+      <a class="nav-link fs-5 active" aria-current="page" href="#">
+        <span>All Jobs</span>
+      </a>
+    </li>
+    <li class="nav-item">
+      <a class="nav-link fs-5" href="#">
+        <span>Public</span>
+      </a>
+    </li>
+    <li class="nav-item">
+      <a class="nav-link fs-5" href="#">
+        <span>Pending</span>
+      </a>
+    </li>
+    <li class="nav-item">
+      <a class="nav-link fs-5" href="#">
+        <span>Processing</span>
+      </a>
+    </li>
+  </ul>
+  <ul class="nav nav-tabs bg-white border border-light border-top-0 ps-7 pe-7">
+    <li class="nav-item">
+      <a class="nav-link active fs-5" href="#">
+        <span>All Jobs</span>
+      </a>
+    </li>
+    <li class="nav-item">
+      <a class="nav-link fs-5" aria-current="page" href="#">
+        <span>Recommended Jobs</span>
+      </a>
+    </li>
+    <li class="nav-item">
+      <a class="nav-link fs-5" href="#">
+        <span>My Favorites</span>
+      </a>
+    </li>
+    <li class="nav-item">
+      <a class="nav-link fs-5" href="#">
+        <span>Applications</span>
+      </a>
+    </li>
+  </ul>
+`;
+
+const tableContent = `
+  <table class="table align-middle fs-5 table-borderless">
```

Comment:
There's some divergence from the design here. We want the table to be edge-to-edge, rather that inset with padding like you have now.

![2021-07-09_10-21-25AM](https://user-images.githubusercontent.com/1605754/125101475-dc936080-e09f-11eb-972e-502439395478.png)